apps/web/pages/api/user/[id].ts

```diff
@@ -11,8 +12,16 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     return res.status(401).json({ message: "Not authenticated" });
   }
 
+  const stringToNumber = z.string().transform((val) => parseInt(val));
+
   const userIdQuery = req.query?.id ?? null;
```

Comment:
Let's just parse the whole req.query shall we?

Reply:
Why userIdQuery can be an array?

Reply:
If don't use an array for anything other that make it "not an array" then we shouldn't support an array.

`?userId=1&userId=2` will produce an array. What should happen here? Should we take the first one? The last one? Should we error?

My personal take is to error rather than allowing to introduce unexpected behavior

Reply:
Totally agree.